structuredefinitions/Extension-UKCore-SpecimenCollectionCollector.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-SpecimenCollectionCollector" />
+  <url value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Specimen.collection.collector" />
+  <version value="1.0.0" />
+  <name value="Extension-UKCore-SpecimenCollectionCollector" />
+  <title value="Extension UK Core Specimen Collection Collector" />
+  <status value="active" />
+  <date value="2023-04-28" />
+  <publisher value="HL7 UK" />
+  <contact>
+    <name value="HL7 UK" />
+    <telecom>
+      <system value="email" />
+      <value value="ukcore@hl7.org.uk" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="Allows the referencing of Patient and RelatedPerson within Specimen.collection." />
+  <purpose value="This is a use case from Genomics to backport R5 functionality extending the referencing of Specimen.collection.collector to include Patient and RelatedPerson" />
+  <fhirVersion value="4.0.1" />
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Specimen.collection" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension.url">
+      <path value="Extension.url" />
+      <fixedUri value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Specimen.collection.collector" />
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]" />
```

Comment:
This need a min value=1 in lline with design approach

Reply:
done